build/storage/scripts/disk_infrastructure.py

```diff
@@ -0,0 +1,207 @@
+import base64
+import importlib
+import logging
+import re
+import socket
+import sys
+import time
+import uuid
+
+from scripts import socket_functions
+
+sys.path.append("/usr/libexec/spdk/scripts")
+sma_client = importlib.import_module("sma-client")
+import rpc
+
+logging.root.setLevel(logging.CRITICAL)
+
+
+def get_number_of_virtio_blk(sock: str) -> int:
+    cmd = 'lsblk --output "NAME"'
+    out = socket_functions.send_command_over_unix_socket(
+        sock=sock, cmd=cmd, wait_for_secs=1
+    )
+    number_of_virtio_blk_devices = len(re.findall("vd", out))
+    return number_of_virtio_blk_devices
+
+
+def is_virtio_blk_attached(sock: str) -> int:
```

Comment:
is there a particular reason why not to return bool?

Reply:
The only reason was to return a return code for bash functions. I changed it to bool.